src/GovUk.Education.ExploreEducationStatistics.Publisher.Model/PublisherQueues.cs

```diff
@@ -4,7 +4,7 @@ public static class PublisherQueues
     {
         public const string GenerateReleaseContentQueue = "generate-release-content";
         public const string PublishAllContentQueue = "publish-all-content";
-        public const string PublishMethodologyQueue = "publish-methodology";
+        public const string PublishMethodologyFilesQueue = "publish-methodology-files";
```

Comment:
Do we just need to make a note at deploy time to ensure there's nothing in the original queue?  And to remove the old queue afterwards?

Reply:
Yes good point, I've started an SOW4 Milestone 1 deploy checklist which I will share. I've added this to it. Thanks.